### PR TITLE
Fixed crash upon destroying PreviewEditor

### PR DESCRIPTION
**Pull request by mooviies**
On my end there was a crash when creating a new file. I investigated it and found out that when destroying PreviewEditor a call to m_customizationDelegate->dispose(); was making aseprite crash.

This is probably because the virtual table is unloaded before dispose is called.

So I added a setCustomizationDelegate(NULL); directly in a new destructor for PreviewEditor which makes the call to dispose correctly before the destructor of Editor is called.

### CODE REVIEW

**Comment by dacap** [comment]
Hi @mooviies, today I'm trying to release a new version so this will wait until next week. Meanwhile could you please [sign the CLA](https://github.com/aseprite/opensource/blob/master/sign-cla.md#sign-the-cla)?

File: src/app/ui/document_view.cpp [hunk 1]
```diff
@@ -160,6 +160,11 @@ class PreviewEditor : public Editor,
     setCustomizationDelegate(this);
   }
 
+  ~PreviewEditor()
+  {
+	setCustomizationDelegate(NULL);
```
Comment (dacap) [inline src/app/ui/document_view.cpp]: It looks like this line is indented with a tab char, we use 2 spaces for indentation.

Comment (mooviies) [inline src/app/ui/document_view.cpp]: Alright I'll fix my settings for 2 spaces indentation

**Comment by dacap** [comment]
Thanks @mooviies, I'll check right now because I was able to reproduce the problem on Windows (release mode only).

**Pull request merged**